<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qCjyxViewControllersModuleWidget</class>
 <widget class="qCjyxWidget" name="qCjyxViewControllersModuleWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>519</width>
    <height>1181</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="ctkCollapsibleButton" name="SliceControllersCollapsibleButton">
     <property name="text">
      <string>Slice Controllers</string>
     </property>
     <layout class="QVBoxLayout" name="SliceControllersLayout"/>
    </widget>
   </item>
   <item>
    <widget class="ctkCollapsibleButton" name="ThreeDViewControllersCollapsibleButton">
     <property name="text">
      <string>3D View Controllers</string>
     </property>
     <layout class="QVBoxLayout" name="ThreeDViewControllersLayout"/>
    </widget>
   </item>
   <item>
    <widget class="ctkCollapsibleButton" name="PlotViewControllersCollapsibleButton">
     <property name="text">
      <string>Plot View Controllers</string>
     </property>
     <layout class="QVBoxLayout" name="PlotViewControllersLayout"/>
    </widget>
   </item>
   <item>
    <widget class="ctkCollapsibleButton" name="AdvancedCollapsibleButton">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Advanced</string>
     </property>
     <property name="collapsed">
      <bool>false</bool>
     </property>
     <layout class="QVBoxLayout" name="SliceInformationLayout">
      <item>
       <layout class="QFormLayout" name="formLayout_2">
        <property name="fieldGrowthPolicy">
         <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
        </property>
        <item row="0" column="1">
         <widget class="qDMMLNodeComboBox" name="DMMLViewNodeComboBox">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="toolTip">
           <string>Select a view from the current scene. Each element corresponds to a specific widget. View nodes are connected with 3D rendering widgets. PlotView with Plotting widgets. Slices (Red, Green, Yellow) with 2D rendering widgets.</string>
          </property>
          <property name="nodeTypes">
           <stringlist>
            <string>vtkDMMLSliceNode</string>
            <string>vtkDMMLViewNode</string>
            <string>vtkDMMLPlotViewNode</string>
           </stringlist>
          </property>
          <property name="showHidden">
           <bool>true</bool>
          </property>
          <property name="addEnabled">
           <bool>false</bool>
          </property>
          <property name="removeEnabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>View Node:</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="qDMMLSliceInformationWidget" name="DMMLSliceInformationWidget"/>
      </item>
      <item>
       <widget class="qDMMLThreeDViewInformationWidget" name="DMMLThreeDViewInformationWidget"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>226</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>qDMMLNodeComboBox</class>
   <extends>QWidget</extends>
   <header>qDMMLNodeComboBox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>qDMMLSliceInformationWidget</class>
   <extends>qDMMLWidget</extends>
   <header>qDMMLSliceInformationWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>qDMMLThreeDViewInformationWidget</class>
   <extends>qDMMLWidget</extends>
   <header>qDMMLThreeDViewInformationWidget.h</header>
  </customwidget>
  <customwidget>
   <class>qDMMLWidget</class>
   <extends>QWidget</extends>
   <header>qDMMLWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>qCjyxWidget</class>
   <extends>QWidget</extends>
   <header>qCjyxWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkCollapsibleButton</class>
   <extends>QWidget</extends>
   <header>ctkCollapsibleButton.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>qCjyxViewControllersModuleWidget</sender>
   <signal>dmmlSceneChanged(vtkDMMLScene*)</signal>
   <receiver>DMMLViewNodeComboBox</receiver>
   <slot>setDMMLScene(vtkDMMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>613</x>
     <y>317</y>
    </hint>
    <hint type="destinationlabel">
     <x>559</x>
     <y>468</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>qCjyxViewControllersModuleWidget</sender>
   <signal>dmmlSceneChanged(vtkDMMLScene*)</signal>
   <receiver>DMMLSliceInformationWidget</receiver>
   <slot>setDMMLScene(vtkDMMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>308</x>
     <y>401</y>
    </hint>
    <hint type="destinationlabel">
     <x>308</x>
     <y>621</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>DMMLViewNodeComboBox</sender>
   <signal>currentNodeChanged(vtkDMMLNode*)</signal>
   <receiver>DMMLSliceInformationWidget</receiver>
   <slot>setDMMLSliceNode(vtkDMMLNode*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>143</x>
     <y>456</y>
    </hint>
    <hint type="destinationlabel">
     <x>130</x>
     <y>489</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>qCjyxViewControllersModuleWidget</sender>
   <signal>dmmlSceneChanged(vtkDMMLScene*)</signal>
   <receiver>DMMLThreeDViewInformationWidget</receiver>
   <slot>setDMMLScene(vtkDMMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>259</x>
     <y>279</y>
    </hint>
    <hint type="destinationlabel">
     <x>259</x>
     <y>454</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>DMMLViewNodeComboBox</sender>
   <signal>currentNodeChanged(vtkDMMLNode*)</signal>
   <receiver>DMMLThreeDViewInformationWidget</receiver>
   <slot>setDMMLViewNode(vtkDMMLNode*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>289</x>
     <y>145</y>
    </hint>
    <hint type="destinationlabel">
     <x>259</x>
     <y>454</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
